frontend/public/components/modals/configure-update-strategy-modal.tsx

```diff
@@ -148,7 +148,50 @@ export const ConfigureUpdateStrategyModal = withHandlePromise((props: ConfigureU
   </form>;
 });
 
+
+export const OLMConfigureUpdateStrategyModal = withHandlePromise((props: OLMConfigureUpdateStrategyModalProps) => {
```

Comment:
This should be defined under frontend/packges/operator-lifecycle-managed

Reply:
> This should be defined under frontend/packges/operator-lifecycle-managed

@spadgett Did you mean under frontend/public/operator-lifecycle-manager?

Reply:
No, it's been restructured and moved to the packages folder.